### PR TITLE
docs - slack update

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Updates our documentation on our Slack integration to incorporate the new Slack App approach.

Related to https://github.com/metabase/metabase/pull/19597.

### CODE REVIEW

File: docs/administration-guide/09-setting-up-slack.md [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
 
-## Setting up Slack
+If you want to have your [Dashboard subscriptions][dashboard-subscriptions] sent to Slack channels (or people on Slack), an admin must first integrate your Metabase with Slack.
 
-If you are using Slack for team communication and would like to have your Pulses be sent to Slack channels (or users) then you'll need to integrate your Metabase instance with Slack.  Luckily, this is really easy!
+Here's an overview of the steps for setting up Slack:
 
-### Generating a Slack API Token
+1. [Create your Slack App](#create-your-slack-app)
+2  [Give Slack your app manifest](#the-app-manifest)
```
Comment (cdeweyx) [inline docs/administration-guide/09-setting-up-slack.md:8]: Seems like this is missing a period and throwing off the ordered list

File: docs/administration-guide/09-setting-up-slack.md [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
 
-Now go to Slack and create a new channel named `metabase_files`. Due to the Slack api, we'll need this to attach graphs to pulses and Metabot answers.
+On the Slack site page for your Slack app, on the left in the **Features** section, click on **OAuth and Permissions** in the Slack Apps sidebar and then copy the **Bot User OAuth Token**. Return to the Slack settings page in your Metabase and paste this token in the Metabase field with the same name.
 
-That's it!  Metabase will automatically run a quick test to check that the API token is working properly and if not you'll get an error message.
+## Create a dedicated Metabase channel in your Slack
+
+In your Slack workspace, create a public channel named whatever you want — we think something like "Metabase" does just fine — then enter that channel's name in the **Slack Channel Name** field in Metabase. This channel allows your Metabase to post to your Slack workspace without having to deal with unnecessary permissions. Due to the way the Slack API is set up, we’ll need this channel to attach charts Dashboard Subscriptions. Make sure the channel you create is the same channel that you enter in this field in Metabase.
```
Comment (cdeweyx) [inline docs/administration-guide/09-setting-up-slack.md:36]: "Metabase" (our suggested channel name) should be lowercase ("metabase") — Slack doesn't let you enter capitalized channel names.

Comment (cdeweyx) [reply docs/administration-guide/09-setting-up-slack.md:36]: "This channel allows your Metabase to post to your Slack workspace without having to deal with unnecessary permissions. Due to the way the Slack API is set up, we’ll need this channel to attach charts Dashboard Subscriptions." 

Could probably be consolidated and I think there's a typo with "attach charts Dashboard Subscriptions"

File: docs/administration-guide/09-setting-up-slack.md [hunk 3]
```diff
@@ -1,33 +1,48 @@
...
 
-That's it!  Metabase will automatically run a quick test to check that the API token is working properly and if not you'll get an error message.
+## Create a dedicated Metabase channel in your Slack
+
+In your Slack workspace, create a public channel named whatever you want — we think something like "Metabase" does just fine — then enter that channel's name in the **Slack Channel Name** field in Metabase. This channel allows your Metabase to post to your Slack workspace without having to deal with unnecessary permissions. Due to the way the Slack API is set up, we’ll need this channel to attach charts Dashboard Subscriptions. Make sure the channel you create is the same channel that you enter in this field in Metabase. 
+
+## Save your changes in Metabase
+
+In Metabase, click on the **Save changes** button and that’s it! Metabase will automatically run a quick test to check that the API token is working properly. If something goes wrong, it'll give you an error message.
```
Comment (cdeweyx) [inline docs/administration-guide/09-setting-up-slack.md:40]: Maybe: "check that the API token and your dedicated Metabase Slack channel are working properly."

Just bc we also check that the Slack channel exists, and if it doesn't we surface an error.

**Review by cdeweyx — COMMENTED** [review]
Left a few small comments but once those are fixed, LGTM 👌

General point, but I agree we can probably do this without adding any images we'll have to maintain.

**Review by cdeweyx — APPROVED** [review]

File: docs/administration-guide/09-setting-up-slack.md [hunk 4]
```diff
@@ -1,33 +1,50 @@
...
+
+## Save your changes in Metabase
+
+In Metabase, click on the **Save changes** button and that’s it! Metabase will automatically run a quick test to check that the API token and your dedicated Slack channel are working properly. If something goes wrong, it'll give you an error message.
 
 ---
 
 ## Next: configuring Metabase
+
 There are a few other settings you configure in Metabase. [Learn how](08-configuration-settings.md).
```
Comment (frannietrempe) [inline docs/administration-guide/09-setting-up-slack.md:48]:
```suggestion
There are a few other settings you configure in Metabase. [Learn how](08-configuration-settings).
```

**Review by frannietrempe — APPROVED** [review]

**Pull request merged**